scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -57,6 +57,18 @@ def execute_command(self, cmd, timeout_secs = TIMEOUT_SECS, verbose = False):
         raise MotrError(ps.returncode, f"\"{cmd}\" command execution failed")
     return stdout, ps.returncode
 
+def execute_command_without_exception(self, cmd, timeout_secs = TIMEOUT_SECS, verbose = False):
+    ps = subprocess.Popen(cmd, stdin=subprocess.PIPE,
+                          stdout=subprocess.PIPE, stderr=subprocess.STDOUT,
+                          shell=True)
+    stdout, stderr = ps.communicate(timeout=timeout_secs);
```

Comment:
subprocess.run does the samething in a single call

Reply:
Corrected..56bbdcf9b02a1b1e618bbd62a446950b92ed77b0